scripts/ci/jobs/upload-db-dump.sh

```diff
@@ -10,6 +10,9 @@ upload_db_dump() {
     if is_in_PR_context; then
         info "Skipping upload, as this is a PR"
     fi
+    if is_tagged; then
```

Comment:
Do I need to ensure this is not master as well, or will this suffice @gavin-stackrox ?

Reply:
It is unclear to me. But from the PR description I think you are trying to deal with inconsistency that can arise between main branch development and patch release development. If this DB artifact does not get used by patch release images or patch release CI then the check is more like `is_a_merge_to_the_main_branch()`